windows/README_MINGW

```diff
@@ -0,0 +1,268 @@
+installation. It was conceived as the starting point for unifying the JACK build
+process across all platforms.
+
+As this is work in progress, there are still a couple of caveats:
+
+- Asynchronous mode is unusable with low latencies
+- JackRouter still builds using a Visual Studio project
+- Lots of real world testing is needed
+
+
+
+Creating the development environment
+------------------------------------
+
+This guide uses MSYS2 as the toolchain, it can be found at https://www.msys2.org/
+It comes as a handy installer called msys2-x86_64-{version}.exe. Once installed:
+
+- Open "MSYS2 MinGW 64-bit terminal" from the MSYS2 start menu shortcuts
+
+- Upgrade base MSYS2 packages
+
+pacman -Suy
+
+It is possible pacman ends with the following banner:
+
+warning: terminate MSYS2 without returning to shell and check for updates again
+warning: for example close your terminal window instead of calling exit
+
+In such case close the MSYS2 window, re-open, and run pacman -Suy again.
+
+- Install required packages
+
+pacman -S mingw-w64-x86_64-toolchain patch autoconf make \
+gettext-devel automake libtool pkgconfig p7zip unzip git python
+
+- Replace the GCC compiler with a version configured for SJLJ exceptions, as
+instructed by the original Windows build instructions (windows/README)
+
+Prebuilt binaries can be found at
+https://sourceforge.net/projects/mingw-w64/files/Toolchains%20targetting%20Win64/
+
+Look for "x86_64-posix-sjlj" under "MinGW-W64 GCC-{version}", the file should be
+called x86_64-{version}-release-posix-sjlj-rt_v6-rev0.7z
+
+Or just download from a direct link (GCC 8.1.0):
+
+wget https://downloads.sourceforge.net/project/mingw-w64/Toolchains%20targetting%20Win64/Personal%20Builds/mingw-builds/8.1.0/threads-posix/sjlj/x86_64-8.1.0-release-posix-sjlj-rt_v6-rev0.7z
+
+Once downloaded:
+
+p7zip -d x86_64-8.1.0-release-posix-sjlj-rt_v6-rev0.7z
+
+That will decompress to a folder called mingw64 in the working directory.
+Now replace the files from the previous mingw-w64-x86_64-toolchain package
+installation:
+
+mv /mingw64 /mingw64.bak   -> backup original
+rm /mingw64                -> see explanation below
+mv mingw64 /               -> sjlj toolchain
+
+The first step results in a new file called mingw64 file being created in /. The
+file is not visible to the Windows File Explorer when inspecting C:\msys64, so
+it is probably an artifact and safe to delete it in the second step. Otherwise
+the last mv step will fail with:
+
+mv: cannot overwrite non-directory '/mingw64' with directory 'mingw64'
+
+An alternate solution consists in closing the MSYS2 window and replicating the
+above procedure using the File Explorer. The following assumes the toolchain 7z
+file was decompressed to the home directory:
+
+Rename C:\msys64\mingw64 to C:\msys64\mingw64.bak
+Move C:\msys64\home\{username}\mingw64 to C:\msys64
+
+Make sure gcc runs and it is the expected version:
+
+$ gcc --version
```

Comment:
I have to put  a path prefix ` /mingw64/bin/` ie ` /mingw64/bin/gcc --version`  should there be an env Path update ?

Reply:
I ran the guide twice and do not recall having to do that. I am not really experienced with MinGW and its build tools, all suggestions are welcomed (continues on next answer)